triton-vm/src/table/master_table.rs

```diff
@@ -1017,26 +1018,27 @@ pub fn terminal_quotient_zerofier_inverse(
     BFieldElement::batch_inversion(zerofier_codeword).into()
 }
 
-/// Computes an array containing all quotients – the Master Quotient Table. Each column corresponds
-/// to a different quotient. The quotients are ordered by category – initial, consistency,
+/// Computes the quotient codeword, which the randomized linear combination of all individual
+/// quotients.
```

Comment:
Modulo max line width:
```suggestion
/// Computes the quotient codeword, which is the randomized linear combination of all individual
/// quotients.
```